maintainers/maintainer-list.nix

```diff
@@ -10496,4 +10496,10 @@
     github = "zupo";
     githubId = 311580;
   };
+  jbcrail = {
```

Comment:
I missed that you did not sort this. Going to fix it in a PR.

Reply:
My apologies, thanks.